src/plot_figs.jl

```diff
@@ -14,18 +12,11 @@ end
 
 function plot_figs(df::DataFrame, path::String)::Nothing
     ENV["GKSwstype"] = "100"  # magic environmental variable for Plots
-    p1 = plot_sol(df, "time", "P+dP")
-    p2 = plot_sol(df, "time", "T")
-    p3 = plot_sol(df, "time", "eps_g")
-    p4 = plot_sol(df, "time", "V")
-    p5 = plot_sol(df, "time", "X_CO2")
-    p6 = plot_sol(df, "time", "total_mass")
-
-    savefig(p1, "$path/1_P.png")
-    savefig(p2, "$path/2_T.png")
-    savefig(p3, "$path/3_eps_g.png")
-    savefig(p4, "$path/4_V.png")
-    savefig(p5, "$path/5_X_CO2.png")
-    savefig(p6, "$path/6_tot_Mass.png")
+    savefig(plot_sol(df, "time", "P+dP"), "$path/1_P.png")
+    savefig(plot_sol(df, "time", "T"), "$path/2_T.png")
+    savefig(plot_sol(df, "time", "eps_g"), "$path/3_eps_g.png")
+    savefig(plot_sol(df, "time", "V"), "$path/4_V.png")
+    savefig(plot_sol(df, "time", "X_CO2"), "$path/5_X_CO2.png")
+    savefig(plot_sol(df, "time", "total_mass"), "$path/6_tot_Mass.png")
```

Comment:
Thank you for providing this good tips! I found if I keep the same name `savefig` for an extra method, it would have errors. So I used `savefig'` instead of `savefig` in the updated code.
```julia
julia> df = chamber(Mafic(), 1e9, 0.2, 0.01, 0.001, -3.3, 8e3)
(thread 1 / 2) Output path: C:\Users\Li-Chen\.julia\dev\Chamber.jl\20230803162943205
number_of_data: 138
ERROR: MethodError: no method matching getindex(::DataFrame, ::Colon, ::Plots.Plot{Plots.GRBackend})
Closest candidates are:
  getindex(::DataFrame, ::Colon, ::Union{Colon, Regex, AbstractVector, DataAPI.All, DataAPI.Between, DataAPI.Cols, InvertedIndices.InvertedIndex}) at C:\Users\Li-Chen\.julia\packages\DataFrames\LteEl\src\dataframe\dataframe.jl:621
  getindex(::DataFrame, ::Colon, ::Union{AbstractString, Signed, Symbol, Unsigned}) at C:\Users\Li-Chen\.julia\packages\DataFrames\LteEl\src\dataframe\dataframe.jl:536
  getindex(::DataFrames.AbstractDataFrame, ::CartesianIndex{2}) at C:\Users\Li-Chen\.julia\packages\DataFrames\LteEl\src\other\broadcasting.jl:3
  ...
Stacktrace:
  [1] plot_sol(df::DataFrame, x_axis_col::String, y_axis_col::Plots.Plot{Plots.GRBackend})    
    @ Chamber C:\Users\Li-Chen\.julia\dev\Chamber.jl\src\plot_figs.jl:3
  [2] (::Chamber.var"#savefig#73"{DataFrame})(label::Plots.Plot{Plots.GRBackend}, path::String) (repeats 2 times)
    @ Chamber C:\Users\Li-Chen\.julia\dev\Chamber.jl\src\plot_figs.jl:16
  [3] _broadcast_getindex_evalf
    @ .\broadcast.jl:670 [inlined]
  [4] _broadcast_getindex
    @ .\broadcast.jl:643 [inlined]
  [5] getindex
    @ .\broadcast.jl:597 [inlined]
  [6] copy
    @ .\broadcast.jl:899 [inlined]
  [7] materialize
    @ .\broadcast.jl:860 [inlined]
  [8] broadcast(::Chamber.var"#savefig#73"{DataFrame}, ::Vector{String}, ::Vector{String})
    @ Base.Broadcast .\broadcast.jl:798
  [9] plot_figs(df::DataFrame, path::String)
    @ Chamber C:\Users\Li-Chen\.julia\dev\Chamber.jl\src\plot_figs.jl:19
 [10] (::var"#2#5"{String, String, Bool, Mafic, Float64, Float64, Float64, Float64, Float64, Float64, IOStream, String, String})()
    @ Main C:\Users\Li-Chen\.julia\dev\Chamber.jl\scripts\runcode.jl:270
```